CMakeLists.txt

```diff
@@ -934,6 +934,7 @@ if (Seastar_DPDK)
   set (Seastar_ARCH_FOR_wsm "westmere")
   set (Seastar_ARCH_FOR_snb "sandybridge")
   set (Seastar_ARCH_FOR_ivb "ivybridge")
+  set (Seastar_ARCH_FOR_hsw "haswell")
```

Comment:
just note for the posterities, dpdk maps hsw to `core-avx2`, see https://github.com/scylladb/dpdk/blob/0f486994114dbc68da07e7345674f7d83b54e0fc/mk/machine/hsw/rte.vars.mk#LL30C25-L30C34

despite that haswell is the first micro architecture which provides avx2 instruction set, gcc has slightly different understanding of these two arches, see https://github.com/gcc-mirror/gcc/blob/26cb95678d42230a8325964a693545723d20005b/gcc/common/config/i386/i386-common.cc#L2034-L2037 . i think the difference only lies in the `model` field, **presumably** it does not change how GCC emits assembly.